pushconfig.sh

```diff
@@ -38,10 +39,12 @@ Available options :
     -V                   Be verbose.
     -d                   Use this script in dry run: no change will be made.
     -h                   Display this help.
+    -F                   Set JSON 'path' to exclude from configuration files. Multiple 'path' are specified using a semi-colon separated list using the jq format.
+    -o                   Override extension of .json configuration file. If set, any file named confXXX.json-<override-extension> will be used instead of confXXX.json.
 EOF
 }
 
-while getopts ":f:a:r:n:s:b:A:u:hvdIHV" opt; do
+while getopts ":f:a:r:n:s:b:A:u:hvdIHVo:" opt; do
```

Comment:
:warning: Why is the `-F` documented before missing in getops and in the following switch ?

Reply:
`-F` c'est un oubli suite à clean-up. Je le dégage